interfaces/builtin/bool_file.go

```diff
@@ -101,6 +101,12 @@ func (iface *BoolFileInterface) SlotSecuritySnippet(plug *interfaces.Plug, slot
 	}
 }
 
+// StaticSlotSecuritySnippet returns the configuration snippet required to provide a bool-file interface.
+func (iface *BoolFileInterface) StaticSlotSecuritySnippet(slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
```

Comment:
Let's please call these "Permanent" rather than "Static".

Let's please have these method names instead:

* `PermanentSlotSnippet`
* `PermanentPlugSnippet`
* `ConnectedSlotSnippet`
* `ConnectedPlugSnippet`